src/daft-connect/src/op/execute/root.rs

```diff
@@ -0,0 +1,72 @@
+use std::{collections::HashMap, future::ready};
+
+use common_daft_config::DaftExecutionConfig;
+use futures::stream;
+use spark_connect::{ExecutePlanResponse, Relation};
+use tokio_util::sync::CancellationToken;
+use tonic::{codegen::tokio_stream::wrappers::UnboundedReceiverStream, Status};
+
+use crate::{
+    op::execute::{ExecuteStream, PlanIds},
+    session::Session,
+    translation,
+};
+
+impl Session {
+    pub async fn handle_root_command(
+        &self,
+        command: Relation,
+        operation_id: String,
+    ) -> Result<ExecuteStream, Status> {
+        use futures::{StreamExt, TryStreamExt};
+
+        let context = PlanIds {
+            session: self.client_side_session_id().to_string(),
+            server_side_session: self.server_side_session_id().to_string(),
+            operation: operation_id,
+        };
+
+        let finished = context.finished();
+
+        let (tx, rx) = tokio::sync::mpsc::unbounded_channel::<eyre::Result<ExecutePlanResponse>>();
```

Comment:
Is unbounded channel necessary?

Reply:
8af7101fea22b350196f1f3d7a0bd6e3155899b1